apps/test/integration/levelSolutions/studio/k1_6.js

```diff
@@ -0,0 +1,138 @@
+var testUtils = require('../../../util/testUtils');
```

Comment:
Seems it was disabled for being flaky in https://github.com/code-dot-org/code-dot-org/commit/12429ab046d6083f8b3fff428bccbfd125cd91c8:

> Disable k1_6 test until we can figure out why it randomly fails

I tried running a few dozen times, and at least locally I had no issues with it.